### ISSUE TITLE
False positive warning is generated for member with doc comments strings.

### ISSUE THREAD

**Issue by MangelMaxime** [fsprojects/FSharpLint#293]
### Description

False positive warning is generated for member with doc comments strings.

### Repro steps

```fs
type Test() =
        /// Convert to self closing element
        member this.ToReactElement(el : IHTMLProp list -> ReactElement): ReactElement =
            // TODO: Remove empty classes?
            let classes = String.concat " " this.Classes |> ClassName :> IHTMLProp
            el (classes::this.Props)
```

There is a warning generated on the doc comment line.

### Expected behavior

Should not generate a warning.

### Actual behavior

Generate a warning: `Class members should be separated by 1 blank line.`

### Known workarounds

None

**Comment by jgardella** [comment]
Hi @MangelMaxime. Thanks for the detailed bug report.

I wasn't able to replicate this; when I lint the provided code I get no errors. Is there any other code surrounding this? Maybe another member following this one? For example:

```
type Test () =
    member this.MemberOne () =
        ()
    /// Doc comment
    member this.MemberTwo () =
        ()
```
